src/components/forms/textarea.jsx

```diff
@@ -17,7 +17,7 @@ var TextArea = React.createClass({
         return (
             <FRCTextarea {... this.props}
                          className={classes}
-                         elementWrapperClassName="textarea-group" />
+                         rowClassName="textarea-row" />
```

Comment:
I think you're still right about `Input` – must've missed that when I was doing the others, I'll do that now